docs/src/api/class-page.md

```diff
@@ -3160,7 +3160,7 @@ Things to keep in mind:
 * The execution time of the handler counts towards the timeout of the action/assertion that executed the handler. If your handler takes too long, it might cause timeouts.
 * You can register multiple handlers. However, only a single handler will be running at a time. Make sure the actions within a handler don't depend on another handler.
 
-:::warning
+:::danger[Warning]
```

Comment:
This one is fresh in my mind, and it was intended to be yellow. So, perhaps, not all of the warning are actually danger. Maybe you were right to keep them as warnings instead?

```suggestion
:::warning[Warning]
```